src/openms/include/OpenMS/ANALYSIS/SEQUENCE/NeedlemanWunsch.h

```diff
@@ -0,0 +1,39 @@
+#include <vector>
+#include <OpenMS/DATASTRUCTURES/String.h>
+#include <OpenMS/OpenMSConfig.h>
+
+
+namespace OpenMS
+{
+  class OPENMS_DLLAPI NeedlemanWunsch
+  {
+
+  public:
+    enum class ScoringMatrix
+    {
+      PAM30MSMatrix,
```

Comment:
IMHO you can remove the `Matrix` part, since its a duplication of effort:
`ScoringMatrix::PAM30MSMatrix` vs `ScoringMatrix:PAM30MS` ??